src/Microsoft.DotNet.Cli.Utils/CommandResolver.cs

```diff
@@ -181,8 +181,7 @@ private static CommandSpec ConfigureCommandFromPackage(string commandName, IEnum
 
                 if (depsPath != null)
                 {
-                    additionalArgs.Add("--depsfile");
-                    additionalArgs.Add(depsPath);
+                    additionalArgs.Add($"--depsfile:{depsPath}");
```

Comment:
Just saw the note about dotnet test.

I didn't realize the host was dependent on args in this format. 
It is a special case out of all the args we pass on the commandline.

Does the host receive de-escaped args like we see in string[] args in Main() ?